specs/ScreenCaptureRequested.md

```diff
@@ -0,0 +1,348 @@
+
+                                    // Get Frame Info
+                                    wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+                                    CHECK_FAILURE(args->get_FrameInfo(&frameInfo));
+
+                                    // Frame Source
+                                    wil::unique_cotaskmem_string frameSource;
+                                    CHECK_FAILURE(frameInfo->get_Source(&frameSource));
+
+                                    // If the host app wants to cancel the request for a specific source
+                                    if (frameSource = "https://developer.microsoft.com/en-us/microsoft-edge/webview2/") 
+                                    {
+                                        CHECK_FAILURE(args->put_Cancel(TRUE));
+                                    }
+
+                                    // Let CoreWebView2 handler know the event is already handled
+                                    CHECK_FAILURE(args->put_Handled(true));
+                                    return S_OK;
+                            })
+                            .Get(),
+                        &m_screenCaptureRequestedToken));
+                }
+                return S_OK;
+        }).Get(),
+        &m_FrameCreatedToken));
+}
+```
+
+## C#: Registering Screen Capture Requested Handler
+```c#
+private WebView2 webView;
+webView.CoreWebView2.ScreenCaptureRequested += (sender, screenCaptureArgs) =>
+{   
+    // Get Frame Info
+    wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+    frameInfo = screenCaptureArgs.frameInfo
+
+    // Frame Source
+    string frameSource;
+    frameSource = frameInfo.Source;
+
+    // If the host app wants to cancel the request from a specific frame
+    if (frameSource == "https://developer.microsoft.com/en-us/microsoft-edge/webview2/")
+    {
+        screenCaptureArgs.Cancel = true;
+    }
+}
+```
+
+
+## C#: Registering IFrame Screen Capture Requested Handler
+```c#
+private WebView2 webView;
+
+m_webview.CoreWebView2.FrameCreated += (sender, frameCreatedArgs) =>
+{
+    // Checking for runtime support of CoreWebView2Frame.ScreenCaptureRequested
+    try
+    {
+        frameCreatedArgs.Frame.ScreenCaptureRequested += (frameSender, screenCaptureArgs) =>
+        {
+            // Get Frame Info
+            wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+            frameInfo = screenCaptureArgs.frameInfo
+
+            // Frame Source
+            string frameSource;
+            frameSource = frameInfo.Source;
+
+            // If the host app wants to cancel the request from a specific source
+            if (frameSource == "https://developer.microsoft.com/en-us/microsoft-edge/webview2/")
+            {
+                screenCaptureArgs.Cancel = true;
+            }
+
+            // Let CoreWebView2 handler know the event is already handled
+
+            // In the case of an iframe requesting permission to use Screen Capture, the default 

```

Comment:
This first part of the comment about the event going from frame to parent is good and should also appear in the Win32 sample code.

Reply:
Accidently created the first one without the markdown extension. Removing the old one. Thanks for catching that!